griptape/tasks/action_subtask.py

```diff
@@ -47,17 +46,33 @@ class ActionSubtask(PromptTask):
     def input(self) -> TextArtifact:
         return TextArtifact(self._input)
 
+    @input.setter
+    def input(self, value: str | TextArtifact | Callable[[BaseTask], TextArtifact]) -> None:
+        if isinstance(value, str):
+            self._input = value
+        else:
+            raise Exception("ActionSubtask input must be a string.")
```

Comment:
I think the signature and the implementation are misleading here

Reply:
Maybe we should just remove this override? It's not really clear to me why it was overridden in the first place.

Reply:
Looks like it'd be fine if we removed the override.

Reply:
Updated. The key part of the override is that we don't render a jinja template. We want the input exactly as it is.